### DISCUSSION TITLE
P10k Prompt Text color question

### DISCUSSION THREAD

**Discussion by J-Siu** [ghostty-org/ghostty#3833] [Q&A] [answered]
I am on MacOS and using P10k prompt. The color/weight look different between ghostty and iTerm2:
![Screenshot 2024-12-28 at 20 32 41](https://github.com/user-attachments/assets/474ed87a-ae42-4e04-b55b-4917944cccc2)

Following is my font config:
```
font-family = 'MesloLGS NF'
font-family-bold = 'MesloLGS NF Bold'
```

Another one is command auto coloring is different too.

Anyone has clue?

**Comment by 00-kat** [comment]
Are you using the same theme in both?  (Ghostty has a theme named `iTerm2 Default` which you can set with `theme = iTerm2 Default`.)

And I don't know how P10k works, but it may be matching on the value of `$TERM` ([someone else](https://discord.com/channels/1005603569187160125/1322400036377727026) had an issue with an oh-my-zsh prompt which was doing this).

**Comment by J-Siu** [comment]
@00-kat Thank you! 'theme = iTerm2 Default' fix the command color issue. But that seems to override lot of stuff in my config. I properly have to create a custom one.

Additionally I discover that single quote mess up the config entry. Removing them from font config as follow partially fixed the prompt:
```
font-family = MesloLGS NF
font-family-bold = MesloLGS NF Bold
```
There are still some inconsistancy with the prompt font weight(bold). Maybe I will deal with them within p10k.zsh.

**Reply by 00-kat** [reply, marked as answer]
> single quote mess up the config entry

Oops, yes, I don't know how I didn't notice that in your original post.  Single quotes don't work, either remove them as you have done, or change them to double quotes.

> There are still some inconsistancy with the prompt font weight(bold).

Check out [`font-thicken`](https://ghostty.org/docs/config/reference#font-thicken).
